spec/models/manageiq/providers/amazon/aws_stubs.rb

```diff
@@ -0,0 +1,228 @@
+            :primary            => true,
+            :association        => {
+              :public_ip     => "58.#{(i / 255) == 0 ? 0 : i % (i / 255)}.#{i / 255}.#{i % 255}",
+              :allocation_id => nil
+            }
+          }, {
+            :private_ip_address => "11.#{(i / 255) == 0 ? 0 : i % (i / 255)}.#{i / 255}.#{i % 255}",
+          }]
+      }
+    end
+
+    {:network_interfaces => ports}
+  end
+
+  def mocked_security_groups
+    mocked_security_groups = []
+
+    test_counts[:security_group_count].times do |i|
+      inbound_firewall_rules = []
+      test_counts[:inbound_firewall_rule_per_security_group_count].times do |fi|
+        inbound_firewall_rules << {
+          :ip_protocol         => "TCP",
+          :from_port           => 0,
+          :to_port             => fi,
+          :user_id_group_pairs => [
+            {
+              :vpc_id => "vpc_#{i}"
+            }]
+        }
+      end
+
+      outbound_firewall_rules = []
+      test_counts[:outbound_firewall_rule_per_security_group_count].times do |fi|
+        outbound_firewall_rules << {
+          :ip_protocol => "TCP",
+          :from_port   => 0,
+          :to_port     => fi,
+          :ip_ranges   => [
+            {
+              :cidr_ip => "0.0.0.0/0"
+            }]
+        }
+      end
+
+      mocked_security_groups << {
+        :group_id              => "security_group_#{i}",
+        :vpc_id                => "vpc_#{i}",
+        :ip_permissions        => inbound_firewall_rules,
+        :ip_permissions_egress => outbound_firewall_rules
+      }
+    end
+    {:security_groups => mocked_security_groups}
+  end
+
+  def mocked_instances
+    instances = []
+    test_counts[:instance_vpc_count].times do |i|
+      instances << {
+        :instance_id        => "instance_#{i}",
+        :network_interfaces => [
+          {
+            :network_interface_id => "interface_#{i}"
+          }]
+      }
+    end
+
+    ec2_max_instances = (test_counts[:instance_vpc_count] + test_counts[:instance_ec2_count] - 1)
+    (test_counts[:instance_vpc_count]..ec2_max_instances).each do |i|
+      instances << {
+        :instance_id => "instance_#{i}"
+      }
+    end
+
+    {:reservations => [{:instances => instances}]}
+  end
+
+  def mocked_vpcs
+    mocked_vpcs = []
+    test_counts[:vpc_count].times do |i|
+      mocked_vpcs << {
+        :vpc_id => "vpc_#{i}"
+      }
+    end
+
+    {:vpcs => mocked_vpcs}
+  end
+
+  def mocked_subnets
+    mocked_subnets = []
+    test_counts[:subnet_count].times do |i|
+      mocked_subnets << {
+        :vpc_id    => "vpc_#{i}",
+        :subnet_id => "subnet_#{i}"
+      }
+    end
+
+    {:subnets => mocked_subnets}
+  end
+
+  def stub_load_balancer_service
+    # TODO(lsmola) seems like Aws doesn't support stubbing of load balancing. Trying to pass stubs using
+    # Aws.config[:ElasticLoadBalancing] fails. We will stub it by rspec for now.
```

Comment:
`Aws.config[:elasticloadbalancing] = { stub_responses => true}` works for me. 
Whats the error?